src/disjoint_set.jl

```diff
@@ -49,6 +49,16 @@ end
 
 find_root(s::IntDisjointSets, x::Integer) = find_root_impl!(s.parents, x)
 
+"""
+    in_same_set(s::IntDisjointSets, x::Integer, y::Integer)
+
+### Args:
+* `s`: An object of type `IntDisjointSets`. 
+* `x`: First of two elements in `s`.
+* `y`: Second element to compare with `x`.
+
+The function returns `true` if `x` and `y` belong to the same subset in `s` and `false otherwise.`
```

Comment:
Ah, and expressions like "the function" are generally not useful: better get straight to the point.